### ISSUE TITLE
Usage with bower in conjunction with svg.select 2.x not possible

### ISSUE THREAD

**Issue by dhardtke** [Fuzzyma/svg.resize.js#13]
Hey,

I can't seem to get this plugin installed when having svg.select.js in version 2 in my bower.json.

This is probably because of the dependency of svg.select.js listed in "dependencies":

```json
 "svg.select.js":"^1.0.4"
```
https://github.com/Fuzzyma/svg.resize.js/blob/master/bower.json

Is there a particular reason for that dependency, or did you, @Fuzzyma only forget to update it?

Thanks.

**Comment by Fuzzyma** [comment]
What's the error message you getting from the install tool? The resize plugin should work with both versions (select 1 and 2) so I am not sure what's the problem is

**Comment by dhardtke** [comment]
You're right - from within my IDE (PhpStorm) I get
`bower                                   ECONFLICT Unable to find suitable version for svg.select.js`

but if I run bower in a dedicated console window I get a prompt letting me choose which version I want to use:

```
Unable to find a suitable version for svg.select.js, please choose one:
    1) svg.select.js#^1.0.4 which resolved to 1.0.5 and is required by svg.resize.js#1.1.0
    2) svg.select.js#~2.0.2 which resolved to 2.0.2 and is required by bla

Prefix the choice with ! to persist it to bower.json
```

Weird.
